Wrappers/Python/cil/utilities/dataexample.py

```diff
@@ -645,4 +356,148 @@ def scikit_random_noise(image, mode='gaussian', seed=None, clip=True, **kwargs):
         if clip:
             out = np.clip(out, low_clip, 1.0)
 
-        return out
\ No newline at end of file
+        return out
+
+    @classmethod
+    def get(cls, data_dir=DEFAULT_DATA_DIR, **load_kwargs):
+        """Calls cls(data_dir).load(cls.dfile, **load_kwargs)"""
+        return cls(data_dir).load(cls.dfile, **load_kwargs)
+
+class BOAT(TestData):
+    dfile = TestData.BOAT
+class CAMERA(TestData):
+    dfile = TestData.CAMERA
+class PEPPERS(TestData):
+    dfile = TestData.PEPPERS
+class RESOLUTION_CHART(TestData):
+    dfile = TestData.RESOLUTION_CHART
+class SIMPLE_PHANTOM_2D(TestData):
+    dfile = TestData.SIMPLE_PHANTOM_2D
+class SHAPES(TestData):
+    dfile = TestData.SHAPES
+class RAINBOW(TestData):
+    dfile = TestData.RAINBOW
+
+class NexusTestData(BaseTestData):
+    @classmethod
+    def get(cls, data_dir=DEFAULT_DATA_DIR):
+        '''
+        Returns
+        -------
+        AcquisitionData
+        '''
+        loader = NEXUSDataReader()
+        loader.set_up(file_name=os.path.join(data_dir, cls.dfile))
+        return loader.read()
+
+class SYNCHROTRON_PARALLEL_BEAM_DATA(NexusTestData):
+    '''A DLS dataset'''
+    dfile = '24737_fd_normalised.nxs'
+class SIMULATED_PARALLEL_BEAM_DATA(NexusTestData):
+    '''A simulated parallel-beam dataset generated from SIMULATED_SPHERE_VOLUME'''
+    dfile = 'sim_parallel_beam.nxs'
+class SIMULATED_CONE_BEAM_DATA(NexusTestData):
+    '''A cone-beam dataset generated from SIMULATED_SPHERE_VOLUME'''
+    dfile = 'sim_cone_beam.nxs'
+class SIMULATED_SPHERE_VOLUME(NexusTestData):
+    '''A simulated volume of spheres'''
+    dfile = 'sim_volume.nxs'
+
+class RemoteTestData(BaseTestData):
+    URL: str
+    FILE_SIZE: str
+
+    @staticmethod
+    def _prompt(msg):
+        while (res := input(f"{msg} [y/n]").lower()) not in "yn":
+            pass
+        return res == "y"
+
+    def download_data(self):
```

Comment:
I think we sometimes want to just download the data without getting it e.g. in the how-to for the readers, the purpose is to show the loading step. In that case I think we need to keep data_dir as an argument and maybe keep this as a class method??